packages/vite/src/node/plugins/css.ts

```diff
@@ -735,44 +737,8 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           (pureCssChunk) => prelimaryNameToChunkMap[pureCssChunk.fileName],
         )
 
-        const emptyChunkFiles = pureCssChunkNames
-          .map((file) => path.basename(file))
-          .join('|')
-          .replace(/\./g, '\\.')
-        const emptyChunkRE = new RegExp(
-          opts.format === 'es'
-            ? `\\bimport\\s*["'][^"']*(?:${emptyChunkFiles})["'];\n?`
-            : `\\brequire\\(\\s*["'][^"']*(?:${emptyChunkFiles})["']\\);\n?`,
-          'g',
-        )
```

Comment:
Sorry I should be a bit succinct on the refactor part. I think we can group this part up as a `function getEmptyChunkReplacer(pureCssChunkNames, opts.format)` that returns a `(str: string) => string` function so that it's more isolated with easily mocked data for unit testing. What do you think?

Reply:
Sounds good. I adjusted the PR.